### PR TITLE
Forward productGroup and lowestPrice fields

### LINKED ISSUE #212: Forward productGroup at the getRecommendations method
SDK can now read & forward the optional value of `"productGroup"` in the raw response that is received from Catalog API.

## Expected Behaviour
Responses from `GET /articles/{configSku}/recommendations` includes a `product_group` string type so it should be transformed to `productGroup` within the implementations of the [getRecommendations method](https://github.com/zalando-incubator/atlas-js-core/blob/master/src/api/atlas_sdk_client.js#L510)

## Current Behaviour
The `product_group` field is not read or forwared.

## Possible Solution
Edit the [RecommendedArticles Model](https://github.com/zalando-incubator/atlas-js-core/blob/master/src/models/recommendation_models.js#L16) to define and include the aforementioned field.

### PR DESCRIPTION
**Pull request by Etchs**
closes #212 and #213

### CODE REVIEW

**Comment by Etchs** [comment]
👍

**Comment by Etchs** [comment]
👍

**Comment by Etchs** [comment]
👍

File: src/models/catalog_api_models.js [hunk 1]
```diff
@@ -26,7 +33,7 @@ const Image = createModel({
 const Video = createModel({
   type: { key: 'type', type: 'string' },
   mediaCharacter: { key: 'mediaCharacter', type: 'string' },
-  url: { key: 'url', type: 'string' }
+  url: { key: 'url', type: 'string', optional: true }
```
Comment (hm6293) [inline src/models/catalog_api_models.js]: Why is the url an optional field ?

Comment (Etchs) [reply src/models/catalog_api_models.js]: Good point. I don't remember why I made this change but I reviewed the updtream services and found that it should be non-optional indeed. It also opened my eyes to some corrupted test data which is also [fixed](https://github.com/zalando-incubator/atlas-js-core/pull/214/commits/b73f9a60c797438409e1a30034e82f9f500f786d) now

**Comment by Etchs** [comment]
👍

File: src/models/catalog_api_models.js (hunk 1 above)
Comment (hm6293) [inline src/models/catalog_api_models.js]: Why is the url an optional field ?

File: src/models/__tests__/recommendation_models_tests.js [hunk 2]
```diff
@@ -0,0 +1,107 @@
+import test from 'ava';
```
Comment (hm6293) [inline src/models/__tests__/recommendation_models_tests.js]: Should we add a test for the optional fields ?

**Comment by Etchs** [comment]
👍

**Comment by hm6293** [comment]
👍

**Pull request merged**